### DISCUSSION TITLE
TidGi 怎样上传图片呢

### DISCUSSION THREAD

**Discussion by stardustman** [tiddly-gittly/TidGi-Desktop#202] [Q&A]
怎样上传图片呢？

**Comment by linonetwo** [comment]
打开编辑模式之后，直接把桌面上的图片拖进文本框里即可弹出导入对话框

**Reply by stardustman** [reply]
Thanks a lot

**Reply by Strelitzia-Reginae-Aiton** [reply]
> 打开编辑模式之后，直接把桌面上的图片拖进文本框里即可弹出导入对话框

markdown类型的tiddler中不能直接插入图片，也不能够以! [图片描述关键词] (图片链接地址)的方式插入本地图片，也不能够以base64的方式插入图片。而TiddlyWiki 5不能像markdown那样渲染较大的latex公式，请问这个问题该如何解决呢？

**Reply by linonetwo** [reply]
tid模式下，katex 插件应该和 Markdown 里的是一样的吧，你遇到啥渲染不出来了？
Markdown对于拖入图片的支持可能得去官方仓库提issue问问，或者去 talk.tiddlywiki.org/ 问问，我最近没咋用 Markdown 模式了

**Reply by Strelitzia-Reginae-Aiton** [reply]
我把tw 5类型的tiddler中没有被渲染的行间公式误认为渲染失败的公式了。我尝试了其他支持tex的插件，发现它们都只支持渲染行间公式。我最后成功通过HTML在tiddler中插入base64格式图片。感谢您的帮助。
